examples/tutorial/04_Machine_Learning.ipynb

```diff
@@ -467,16 +425,7 @@
    "metadata": {},
    "outputs": [],
    "source": [
-    "import geoviews.tile_sources as gts"
-   ]
-  },
-  {
-   "cell_type": "code",
-   "execution_count": null,
-   "metadata": {},
-   "outputs": [],
-   "source": [
-    "gts.ESRI * unstacked.hvplot(x='x', y='y', datashade=True, geo=True, height=500, cmap='Category10')"
+    "gv.tile_sources.ESRI * unstacked.hvplot(x='x', y='y', geo=True, height=500, cmap='Category10', alpha=0.7)"
```

Comment:
I think ESRI is deprecated, and should be something more specific (e.g. EsriImagery?)